actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -162,6 +162,8 @@ private void createUserV3_V4(Request actorMessage, boolean isV4) {
       if (!(StringUtils.isNotBlank(managedBy))) {
         userService.validateUserId(actorMessage, managedBy);
       }
+      //If managedUser limit is set, validate total number of managed users against it
+      UserUtil.validateManagedUserLimit(managedBy);
```

Comment:
could this validation be inside the if clause above?

Reply:
Done!